UnityProject/Assets/Scripts/Messages/MessageTypes.cs

```diff
@@ -15,6 +15,7 @@ enum MessageTypes : short
     UpdateUIMessage = 1006,
     ClosetHandlerMessage = 1007,
     ForceJobListUpdateMessage = 1008,
+    PlayerDeathMessage = 1009,
```

Comment:
You can already prepare yourself for CNT and move it to 1010
Because I will merge this after CNT anyway